src/views/Auth.vue

```diff
@@ -1,35 +1,34 @@
 <template>
-  <Login v-if="toggle" @toggleAuth="toggleAuth" @login="handleLogin"/>
-  <Signup v-else @toggleAuth="toggleAuth" @login="handleLogin"/>
+	<Login v-if="toggle" @toggleAuth="toggleAuth" @login="handleLogin" />
+	<Signup v-else @toggleAuth="toggleAuth" @login="handleLogin" />
 </template>
 
 <script>
-import Login from "@/components/Login";
-import Signup from "../components/Signup";
-import {ref} from "vue";
-import {useRouter} from "vue-router";
+import Login from '@/components/Login'
+import Signup from '../components/Signup'
+import { ref } from 'vue'
+import { useRouter } from 'vue-router'
+import ToastComponent from '../components/ToastComponent.vue'
 
 export default {
-  name: "Auth",
-  components: {Signup, Login},
-  setup() {
-    const toggle = ref(true);
+	name: 'Auth',
+	components: { Signup, Login, ToastComponent },
```

Comment:
Remove the import here, as you were doing it in App.vue

Reply:
done!

Reply:
done!